doc/old_and_new_proposal.md

```diff
@@ -647,6 +650,14 @@ following options.
 
   * `lvm`
     Whether LVM should be used by default.
+  * `separate_vgs`
+    Whether every volume specifying a separate_vg_name should be created as
+    isolated LVM Volume Group instead include them in the "system" group.
+  * `allocate_volume_mode`
```

Comment:
Too many commas in this paragraph. Something I'm learning to avoid. On the other hand, we should mention something about the impact in the UI.

When set to `auto`, the proposal expects a set of candidate disks in which it will distribute the volumes automatically as needed. If set to `device`, the proposal needs to know in which disk must place each volume. As a consequence, the interface presented to the user by default will be different (containing different questions) depending of the value of this property.